### PR TITLE
Adding  some explanation about metadata integrity and authenticity

### PR DESCRIPTION
**Pull request by riccardoAlbertoni**
This is an early attempt to include the feedback from #1526 in the DCAT. 

PREVIEW: 
https://raw.githack.com/w3c/dxwg/DCAT-issue-1526-bis/dcat/index.html#security_and_privacy

Diff:
https://services.w3.org/htmldiff?doc1=https://w3c.github.io/dxwg/dcat/&doc2=https://raw.githack.com/w3c/dxwg/DCAT-issue-1526-bis/dcat/index.html#security_and_privacy

### CODE REVIEW

**Comment by riccardoAlbertoni** [comment]
Thanks, @agreiner, for complementing the proposal.

I have added a draft link sentence to verifiable credentials. 
Please, @pchampin, check if that sentence mirrors your suggestions.

File: dcat/index.html [hunk 1]
```diff
@@ -5978,6 +5978,8 @@ <h2>Security and Privacy</h2>
...
...operty <a href="#Property:distribution_checksum">spdx:checksum</a> from [[!SPDX]] to ensure the integrity and authenticity of DCAT distributions. It is worth noting that the associated checksum will not provide the expected security protections if the integrity or authenticity of the DCAT metadata is also not guaranteed. Integrity and authenticity of DCAT metadata depend on the trustworthiness of the source. DCAT providers should address integrity and authenticity at the application level. For example, they should ensure the integrity and authenticity of their API and download endpoints and make DCAT metadata files downloadable from authoritative origins. DCAT does not prescribe the manner of generating the checksum. Publishers should provide the necessary detail for the user to reliably calculate the provided hash from the files supplied. Development of a canonical method for the generation of a checksum overlaps with the scope of the <a href="https://www.w3.org/groups/wg/rch">RDF Dataset Canonicalization and Hash Working Group</a>, on which DCAT can build in the future. Whereas, Verifiable Credentials [[?VC-DATA-MODEL-2.0]] can be explored when DCAT metadata is served as JSON-LD.
```
Comment (andrea-perego) [inline dcat/index.html]:
```suggestion
<p>DCAT borrows the property <a href="#Property:distribution_checksum"><code>spdx:checksum</code></a> from [[!SPDX]] to ensure the integrity and authenticity of DCAT distributions. It is worth noting that the associated checksum will not provide the expected security protections if the integrity or authenticity of the DCAT metadata is also not guaranteed. Integrity and authenticity of DCAT metadata depend on the trustworthiness of the source. DCAT providers should address integrity and authenticity at the application level. For example, they should ensure the integrity and authenticity of their API and download endpoints and make DCAT metadata files downloadable from authoritative origins. DCAT does not prescribe the manner of generating the checksum. Publishers should provide the necessary detail for the user to reliably calculate the provided hash from the files supplied. Development of a canonical method for the generation of a checksum overlaps with the scope of the <a href="https://www.w3.org/groups/wg/rch">RDF Dataset Canonicalization and Hash Working Group</a>, on which DCAT can build in the future. Moreover, the use of Verifiable Credentials [[?VC-DATA-MODEL-2.0]] can be explored when DCAT metadata is served as JSON-LD.
```

**Review by andrea-perego — COMMENTED** [review]
Some suggested editorial fixes.

**Comment by riccardoAlbertoni** [comment]
> Some suggested editorial fixes.

Thanks a lot, @andrea-perego.
I have accepted your suggestions and added a new line to the change history in the document.

**Review by davebrowning — APPROVED** [review]

**Review by pchampin — APPROVED** [review]

**Pull request merged**